common/src/commonMain/kotlin/io/xorum/codeforceswatcher/features/auth/redux/AuthReducer.kt

```diff
@@ -46,7 +46,7 @@ fun authReducer(action: Action, state: AppState): AuthState {
         is AuthRequests.UpdateAuthStage -> {
             newState = newState.copy(authStage = action.authStage)
         }
-        is AuthRequests.LogOut -> {
+        is AuthRequests.LogOut.Success -> {
```

Comment:
Is it possible to get `Failure` over here?

Reply:
It's possible on iOS